src/navigation/components/Header/Header.tsx

```diff
@@ -52,11 +57,32 @@ const MobileMenuButton: React.FC<MobileMenuButtonProps> = ({
     );
 };
 
+const iconSizeKey: keyof NavigationItemBase = 'iconSize';
+
 export const Header: React.FC<HeaderProps> = ({data, logo}) => {
-    const {leftItems, rightItems} = data;
+    const {leftItems, rightItems, iconSize} = data;
     const [isSidebarOpened, setIsSidebarOpened] = useState(false);
     const [activeItemId, setactiveItemId] = useState<string | undefined>(undefined);
 
+    const getNavigationItemWithIconSize = useCallback(
+        (item: NavigationItemModel) => {
+            if (!(iconSizeKey in item)) {
+                return {...item, iconSize};
```

Comment:
What happens if iconSize is not set? Maybe you should add the default value

Reply:
If iconSize is not set, then no styles will be applied to the image.
seeIconSizeStyle: https://github.com/gravity-ui/page-constructor/pull/329/files#diff-01cc3be42ecfb8f27579002a8ce8d95d1aa53569e00008dd1510400ed43425e7R18

Reply:
Maybe we should set some default size?